packages/web/src/js/utils/ComponentFunctions.ts

```diff
@@ -53,4 +53,14 @@ const enableToggleAutoloader = (component: typeof BaseComponent, method = 'toggl
 
 const clickOutsideElement = (target: Element, event: Event) => !event.composedPath().includes(target);
 
-export { enableToggleTrigger, enableDismissTrigger, enableToggleAutoloader, clickOutsideElement };
+const debounce = (func: () => void, time: number) => {
```

Comment:
We were talking about the debounce and setTimeout are not needed. What has changed?

Reply:
I thought we agreed to solve this in a separate task

Reply:
I modified it as it is in BS, added a few helper methods for that handling and threw out the resize handler and moved those timeouts as async. Also removed 'debounce' method.